www/proxy-sso/src/opnsense/service/templates/OPNsense/ProxySSO/krb5.conf

```diff
@@ -0,0 +1,38 @@
+# Autogenerated config. Do not edit manualy.
+
+{% set ldap = [] %}
+{% if helpers.exists('OPNsense.proxy.forward.authentication.method') %}
+{%      for method in OPNsense.proxy.forward.authentication.method.split(",") %}
+{%          for server in helpers.toList('system.authserver') %}
+{%              if server.type == 'ldap' and server.name == method %}
+{%                      if ldap.append(server) %}{% endif %}
```

Comment:
empty if?

Reply:
Yes, it is a way to provide value changes from inner loop to outer variable (ldap) in jinja2.
ldap.append(server) makes it working.

Reply:
Yes you are right.